src/components/Header/Header.js

```diff
@@ -21,7 +20,8 @@ const HeaderWrapper = styled.header`
 const AppName = styled(NavLink)`
   color: ${colors.black};
   font-family: Satisfy;
-  font-size: 8vw;
+  font-size: 3em;
```

Comment:
for next PR. 

I think, that you should use rem. I remember that I was talking shits about that very long time ago to you. 
Imo, font-sizes could be in some constants (like textLarge, or textL, or something like this).